routes/index.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import fetch from 'node-fetch';
+
+var router = express.Router();
+
+/* GET home page. */
+router.get('/', async function(req, res, next) {
+
+  const statusUrl = "http://localhost:8000/publishers/status";
```

Comment:
The base URL part (eg "http://localhost:8000") should be set as some kind of global variable for re-use and taken from ENV - I can't see there would be any technical blocker on that?

But also we could do it later, so I don't think it has to block this P.R.